src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer.Common/RazorCustomLSPMethodNames.cs

```diff
@@ -29,7 +35,9 @@ internal static class RazorLanguageServerCustomMessageTargets
     public const string RazorHtmlFormattingEndpoint = "razor/htmlFormatting";
     public const string RazorHtmlOnTypeFormattingEndpoint = "razor/htmlOnTypeFormatting";
 
-    // Still to migrate
+    // VS Windows only at the moment, but could/should be migrated
+    public const string RazorDocumentSymbolEndpoint = "razor/documentSymbol";
```

Comment:
Should we create issues for tracking adding this (and others) to VS Code?

Reply:
This is probably a good thing to bring up at a sync meeting. Being in this list doesn't mean that its not supported in VS Code, as there are a bunch of these that are supported in VS Code using the VS Code API and not LSP. We probably need a proper audit, and to come up and agree on a plan.